### PR TITLE
add PHP 7.4

### PR DESCRIPTION
**Pull request by aaronjorbin**
It's not stable, but it should be one day.

### CODE REVIEW

File: update.php [hunk 1]
```diff
@@ -137,6 +137,19 @@
 			'download_url' => 'https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar',
 		),
 	),
+	'7.4' => array(
+		'php' => array(
+			'base_name'       => 'php:7.4-rc-fpm',
+			'gd'              => true,
+			'extensions'      => array( 'gd', 'opcache', 'mysqli', 'zip' ),
+			'pecl_extensions' => array( 'xdebug-2.7.2' ),
```
Comment (pento) [inline update.php]:
```suggestion
			'pecl_extensions' => array( 'xdebug-2.8.0beta1' ),
```

Xdebug 2.8.0beta1 is required for PHP 7.4 support.

File: 7.4/php/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
+FROM php:7.4-rc-fpm
```
Comment (ntwb) [inline 7.4/php/Dockerfile]:
```suggestion
FROM php:7.4.0beta1-fpm
```

Based on the [comment below](https://github.com/WordPress/wpdev-docker-images/pull/1/files#r310881090) using `xdebug-2.8.0beta1` 2.8

Via https://xdebug.org/

> _"This first beta release makes Xdebug compile with the PHP 7.4beta1 release, and also fixes a handful of other bugs."_


I'd suggest also using that cited build of PHP 7.4, what the difference between `7.4.0beta1-fpm` and `7.4-rc-fpm` actually are, I do not know, but at this point in time an RC is not scheduled until 5th September https://wiki.php.net/todo/php74

File: update.php [hunk 3]
```diff
@@ -137,6 +137,19 @@
 			'download_url' => 'https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar',
 		),
 	),
+	'7.4' => array(
+		'php' => array(
+			'base_name'       => 'php:7.4-rc-fpm',
```
Comment (ntwb) [inline update.php]:
```suggestion
			'base_name'       => 'php:7.4.0beta1-fpm',
```

See [existing comment](https://github.com/WordPress/wpdev-docker-images/pull/1/files#r310900656)

File: 7.4/php/Dockerfile (hunk 2 above)
Comment (pento) [reply 7.4/php/Dockerfile]: Tests are passing against 7.4snapshot, so it should be fine to use RC.

https://travis-ci.org/xdebug/xdebug/jobs/567376266

**Pull request merged**